Duplicati/Library/Common/IO/SystemIOWindows.cs

```diff
@@ -496,31 +497,44 @@ public void SetMetadata(string path, Dictionary<string, string> data, bool resto
             var isDirTarget = path.EndsWith(DIRSEP, StringComparison.Ordinal);
             var targetpath = isDirTarget ? path.Substring(0, path.Length - 1) : path;
 
-            FileSystemSecurity rules = isDirTarget ? GetAccessControlDir(targetpath) : GetAccessControlFile(targetpath);
-            if (restorePermissions && data.ContainsKey("win-ext:accessrules"))
+            if (restorePermissions)
             {
-                var content = DeserializeObject<FileSystemAccess[]>(data["win-ext:accessrules"]);
-                var c = rules.GetAccessRules(true, false, typeof(System.Security.Principal.SecurityIdentifier));
-                for (var i = c.Count - 1; i >= 0; i--)
-                    rules.RemoveAccessRule((System.Security.AccessControl.FileSystemAccessRule)c[i]);
-
-                Exception ex = null;
+                FileSystemSecurity rules = isDirTarget ? GetAccessControlDir(targetpath) : GetAccessControlFile(targetpath);
 
-                foreach (var r in content)
+                if (data.ContainsKey("win-ext:accessrulesprotected"))
                 {
-                    // Attempt to apply as many rules as we can
-                    try
+                    bool IsProtected = bool.Parse(data["win-ext:accessrulesprotected"]);

```

Comment:
We typically begin variable names with a lowercase letter.

Reply:
Ok, that's a simple change.